Sources/CacheAdvance/CacheReader.swift

```diff
@@ -23,9 +23,12 @@ final class CacheReader {
 
     /// Creates a new instance of the receiver.
     ///
-    /// - Parameter file: The file URL indicating the desired location of the on-disk store. This file should already exist.
-    init(forReadingFrom file: URL) throws {
+    /// - Parameters:
+    ///   - file: The file URL indicating the desired location of the on-disk store. This file should already exist.
+    ///   - maximumBytes: The maximum size of the cache, in bytes. Logs larger than this size will fail to append to the store.
```

Comment:
Let's update this comment. We're in a reader so we're not appending in this class.